pkg/migrate/migrate.go

```diff
@@ -1146,3 +1296,249 @@ func resetReclaimPolicy(ctx context.Context, w *log.Logger, clientset k8sclient.
 
 	return nil
 }
+
+// buildPVConsumerPod creates a pod spec for consuming a pvc
+func (p *PVMigrator) buildPVConsumerPod(pvcName string) *corev1.Pod {
```

Comment:
can we pass in options instead of using an object?
It's not the patten the project has used thus far.

Reply:
@laverya you want `buildPVConsumerPod()` to accept the name override as an argument instead of pulling it from the receiver?